aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/plugin/util/PluginReflectionUtility.java

```diff
@@ -332,11 +333,11 @@ private static void populateRuntimeContext(Object handler, Field field) {
             .stream()
             .collect(Replacer.processSourceReplace());
 
-        return reducedWithReplacements
+        return OrderingUtil.sortMembers(reducedWithReplacements
```

Comment:
It's better not to enclose the whole of 4 lines of stream into single brackets. Makes readability much worse. You would rather go `List<Source> preSortedByRank = reducedWithReplacements.stream()....; return OrderingUtils.sortMembers(preSortedByRank)` (you actually had similar notation in one of the previous commits)